common/build/eslint-config-fluid/README.md

```diff
@@ -32,8 +32,6 @@ a diff to review as part of a PR -- just like we do with API reports for code ch
 <!-- AUTO-GENERATED-CONTENT:START (SCRIPTS) -->
 | Script | Description |
 |--------|-------------|
-| `build` | Build the package. |
-| `build:ci` | Build the package -- called as part of CI. |
```

Comment:
This and other markdown file changes are because I did a local docs build and some of the files were outdated.